_news/2020-12-17-v6.2.0-released.md

```diff
@@ -51,7 +51,7 @@ Users can then model whether an operation is a _read-only_ or a _write_ operatio
   }
 </pre></div>
 
-More details about the new syntax for Aggregate lifecycles and state transitions can be found [here](/docs/aggregate/#aggregate-lifecycle-and-state-transitions) on the [Aggregate page of the language reference](/docs/aggregate/#aggregate-lifecycle-and-state-transitions).
+More details about the new syntax for Aggregate lifecycles and state transitions can be found [here](/docs/aggregate/#aggregate-lifecycle-and-state-transitions) on the [Aggregate page of the language reference](/docs/aggregate/). 
```

Comment:
Why removing that anchor? With the anchor the user does not have to scroll down...